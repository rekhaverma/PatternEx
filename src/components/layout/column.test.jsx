import React from 'react';
import Column from './column.component';

/**
 * @status: Complete
 * @sign-off-by: Alex Andries
 */
describe('<Column />', () => {
  it('should match with snapshot', () => {
    const props = {
      children: <div />,
    };
    const wrapper = shallow(<Column {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
