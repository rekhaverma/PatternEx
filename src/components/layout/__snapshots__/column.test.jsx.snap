// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Column /> should match with snapshot 1`] = `
<div
  className="column__1__1 "
  style={Object {}}
>
  <div />
</div>
`;
